en/docs/learn/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -34,329 +34,328 @@ The following concepts and artifacts are used in this tutorial:
+        </td>
+      </tr>
+      <tr>
+        <td>URL Style</td>
+        <td>
+          <code>URI_TEMPLATE</code>
+        </td>
+      </tr>
+      <tr>
+        <td>Methods</td>
+        <td>
+          <code>GET</code> <br> This defines that the API resource only handles requests where the HTTP method is GET.
+        </td>
+      </tr>
     </table>
 
-    !!! Tip
-        See the [Service Catalog API documentation](https://apim.docs.wso2.com/en/4.2.0/reference/product-apis/service-catalog-apis/service-catalog-v1/service-catalog-v1/) for more information on the metadata in the YAML file.
+    <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/edit-api-resource.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/edit-api-resource.png" alt="edit API resource" width="40%"></a>
 
-3.  <b>Important</b>: Be sure to change the `serviceUrl` from HTTPS to HTTP. This is required because the HealthcareAPI is not secured.
+8. Click **Update**.
 
-4.  Leave the default values for the remaining parameters. 
+#### Create the mediation logic
 
-### Step 3 - Configure the Micro Integrator 
+You can now configure the mediation logic to handle requests.
 
-The Micro Integrator contains a client application, which automatically publishes artifacts to the **Service Catalog** in the **API Publisher** portal. 
+1. To get started, click on the **+** icon to add the first mediator to the sequence.
 
-Let's enable this client for the embedded Micro Integrator of WSO2 Integration Studio.
+    <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/add-log.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/add-log.png" alt="add log" width="80%"></a>
 
-1.  Click the <b>Embedded Micro Integrator Configuration</b> (<a href="{{base_path}}/assets/img/integrate/tutorials/common/server-config-64x64.png"><img src="{{base_path}}/assets/img/integrate/tutorials/common/server-config-64x64.png" width="20"></a>) icon on the upper menu to open the dialog box.
-2.  Uncomment the `[[service_catalog]]` section as shown below and change the APIM server configurations accordingly. 
+2. Select **Log** mediator in the **Generic** section under **All Mediators**.
 
-    !!! Tip
-        The default username and password for connecting to the API gateway is `admin`.
+    <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/log-mediator.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/log-mediator.png" alt="log mediator" width="30%"></a>
 
-    ```toml
-    [[service_catalog]]
-    apim_host = "https://localhost:9443"
-    enable = true
-    username = "admin"
-    password = "admin"
-    ```
+    !!! Note
```

Comment:
Should this be a note? Can we add in the same level

Reply:
Fixed with [12fb9cf](https://github.com/wso2/docs-mi/pull/984/commits/12fb9cff24e485d2451dd6e40e7502db7849dae2)